roles/ceph-common/tasks/main.yml

```diff
@@ -231,10 +231,35 @@
     line: "CLUSTER={{ cluster }}"
   when: ansible_os_family == "RedHat"
 
+# NOTE(leseb): we are performing the following check
+# to ensure any Jewel installation will not fail.
+# The following commit https://github.com/ceph/ceph/commit/791eba81a5467dd5de4f1680ed0deb647eb3fb8b
+# fixed a package issue where the path was the wrong.
+# This bug is not yet on all the distros package so we are working around it
```

Comment:
Please add https://bugs.launchpad.net/ubuntu/+source/ceph/+bug/1582773 

I'm not sure about Infernalis from Ceph.com, but Hammer from ceph.com is not affected. This is why we've got to have confirmed tickets.